src/linear_algebra/affine_space/finite_dimensional.lean

```diff
@@ -68,7 +68,7 @@ instance finite_dimensional_direction_affine_span_image_of_finite [_root_.finite
 finite_dimensional_direction_affine_span_of_finite k (set.to_finite _)
 
 /-- An affine-independent family of points in a finite-dimensional affine space is finite. -/
-lemma finite_of_fin_dim_affine_independent [finite_dimensional k V] {p : ι → P}
+lemma finite_of_fin_rank_affine_independent [finite_dimensional k V] {p : ι → P}
```

Comment:
Looks like I need to do a cleanup pass replacing `fin_rank` (not to be confused with `finrank`) back with `fin_dim`.

Reply:
The build passed the first round, so I've pushed an update with this change.